ArduPlane/Parameters.h

```diff
@@ -265,17 +265,17 @@ class Parameters {
         k_param_throttle_fs_value,
         k_param_throttle_cruise,
 
-        k_param_short_fs_action,
-        k_param_long_fs_action,
+        k_param_fs_action_short,
+        k_param_fs_action_long,
         k_param_gcs_heartbeat_fs_enabled,
         k_param_throttle_slewrate,
         k_param_throttle_suppress_manual,
         k_param_throttle_passthru_stabilize,
         k_param_rc_12_old,
         k_param_fs_batt_voltage,
         k_param_fs_batt_mah,
-        k_param_short_fs_timeout,
-        k_param_long_fs_timeout,
+        k_param_fs_timeout_short,
+        k_param_fs_timeout_long,
```

Comment:
double sanity checking here.. This rename of k_param_* does not matter as long as the order is not changed, right?

Reply:
yeah, that's right.  the rename isn't going to cause corruption.